data-new/BuildAndDeployment/Sub-Dimensions.yaml

```diff
@@ -0,0 +1,398 @@
+---
+_meta:
```

Comment:
now we can add further _meta info to dimensions: this will enable to better describe the dimensions and add new things.